plugins/Google/README.md

```diff
@@ -1,2 +1,27 @@
 This is a simple plugin to provide access to the Google services we all know
 and love from our favorite IRC bot.
+
+Commands: cache, calc, fight, google, lucky, phonebook, and translate
+
+Basic usage
+-----------
+
+1. google
+ 
+ Searches for a string and gives you 3 results from google search
+ >!google something
+ 
+2. lucky
+ 
+ Return the first result (Google lucky search)
+ >!lucky something
+
+3. calc
+ 
+ Does mathematic calculations
+ >!calc 5+4
+
+4. translate
+
+ Translates a string
```

Comment:
it might be helpful to note which language codes are supported here: https://cloud.google.com/translate/v2/using_rest#language-params